types/steamapi/index.d.ts

```diff
@@ -0,0 +1,195 @@
+// Type definitions for steamapi 2.1
+// Project: https://github.com/xDimGG/node-steamapi
+// Definitions by: vanitasboi <https://github.com/vanitasboi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class SteamAPI {
+        constructor(key: string, options?: ConstructorOptions);
```

Comment:
```suggestion
        constructor(key: string, options?: Options);
```
Just `Options` would align better with api and readme